src/pages/ChatMainPage/chat-main-page.module.css

```diff
@@ -125,9 +125,21 @@ select:focus {
   border: 1px solid var(--secondary-background-illustration, #d7d2e6);
   padding: 10px;
 }
+.aside_input_wrap_mobile {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 350px;
+  /* margin-bottom: 20px; */
+
+  border-bottom: 1px solid var(--secondary-background-illustration, #d7d2e6);
```

Comment:
Why you can't use only var here and move #d7d2e6 to var initialization?

Reply:
Just copied from figma